lib/duo_sig.js

```diff
@@ -72,6 +72,18 @@ function canonicalize (method, host, path, params, date) {
   ].join('\n')
 }
 
+function canonicalizeV5 (method, host, path, params, date, body) {
+  return [
+    date,
+    method.toUpperCase(),
+    host.toLowerCase(),
+    path,
+    canonParams(params),
+    hashString(body),
+    hashString('') // additional headers not needed at this time
+  ].join('\n')
+}
+
 // Return the Authorization header for an HMAC signed request.
 function sign (ikey, skey, method, host, path, params, date) {
   var canon = canonicalize(method, host, path, params, date)
```

Comment:
No need to change the method name, but for clarity for future Duo devs, can you add a comment or something that this is called "V2" historically?  otherwise the "V5" above is somewhat inexplicable

ditto on the canonicalization method also